stubs/inertia/resources/js/Components/Dropdown.vue

```diff
@@ -65,9 +65,8 @@ const alignmentClasses = computed(() => {
         >
             <div
                 v-show="open"
-                class="absolute z-50 mt-2 rounded-md shadow-lg"
+                class="absolute hidden z-50 mt-2 rounded-md shadow-lg"
                 :class="[widthClass, alignmentClasses]"
-                style="display: none;"
```

Comment:
The `hidden` class will prevent `v-show` from working so you'll never be able to see the dropdown.

I'm not sure why we needed the inline style though - I'd assume it could just be removed?